xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/main/resources/XWiki/Lightbox/WebHome.xml

```diff
@@ -0,0 +1,655 @@
+      aria-keyshortcuts="Space" aria-pressed="false"
+    &gt;$services.icon.renderHTML('play')&lt;/a&gt;
+    &lt;a
+      class="btn escape" aria-controls="blueimp-gallery"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.escape.title'))"
+      title="$escapetool.xml($services.localization.render('image.lightbox.escape.title'))"
+      aria-keyshortcuts="Escape"
+    &gt;$services.icon.renderHTML('cross')&lt;/a&gt;
+  &lt;/div&gt;
+  &lt;div class="lightboxDescription"&gt;
+    &lt;div class="descriptionCaption"&gt;
+      &lt;h3 class="caption"
+        aria-label="$escapetool.xml($services.localization.render('image.lightbox.caption.label'))"&gt;&lt;/h3&gt;
+    &lt;/div&gt;
+    &lt;div class="descriptionMetadata"&gt;
+      &lt;div class="descriptionRow"&gt;
+        &lt;span class="title"
+          aria-label="$escapetool.xml($services.localization.render('image.lightbox.fileName.label'))"&gt;&lt;/span&gt;
+      &lt;/div&gt;
+      &lt;div class="descriptionRow"&gt;
+        &lt;span class="publisher"
+          aria-label="$escapetool.xml($services.localization.render('image.lightbox.publisher.label'))"&gt;&lt;/span&gt;
+        &lt;span class="date"
+          aria-label="$escapetool.xml($services.localization.render('image.lightbox.date.label'))"&gt;&lt;/span&gt;
+      &lt;/div&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+  &lt;ol class="indicator"&gt;&lt;/ol&gt;
+&lt;/div&gt;
+&lt;!-- The image toolbar popover template. --&gt;
+&lt;div id="imageToolbarTemplate"&gt;
+  &lt;div class="btn-group imageToolbar" hidden&gt;
+    &lt;a id="openLightbox"
+      title="$escapetool.xml($services.localization.render('image.lightbox.gallery.title'))"&gt;
+      $services.icon.renderHTML("arrows")
+    &lt;/a&gt;
+    &lt;a class="imageDownload" target="_blank"
+      title="$escapetool.xml($services.localization.render('image.lightbox.download.title'))"&gt;
+      $services.icon.renderHTML("download")
+    &lt;/a&gt;
+  &lt;/div&gt;
+&lt;/div&gt;
+#end
+#set ($lightboxConfigDoc = $xwiki.getDocument('XWiki.Lightbox.LightboxConfiguration'))
+#set ($lightboxConfigObj = $lightboxConfigDoc.getObject('XWiki.Lightbox.LightboxConfigurationClass'))
+#if ($lightboxConfigObj.getValue('isLightboxEnabled').equals(1))
+  #set ($discard = $xwiki.linkx.use($services.webjars.url('org.xwiki.contrib:webjar-blueimp-gallery',
+    'css/blueimp-gallery.min.css'), {'type': 'text/css', 'rel': 'stylesheet'}))
+  #set ($discard = $xwiki.linkx.use($services.webjars.url('org.xwiki.platform:xwiki-platform-image-webjar',
+    'less/lightbox.less', {'evaluate': true}), {'type': 'text/css', 'rel': 'stylesheet'}))
+  #set ($discard = $xwiki.jsx.use('XWiki.Lightbox.WebHome'))
+  #set ($config = {})
+  #set ($discard = $config.put('HTMLTemplate', "#lightboxHTMLTemplate"))
+  {{html clean="false"}}
+  &lt;script id="lightbox-config" type="application/json"&gt;$jsontool.serialize($config)&lt;/script&gt;
```

Comment:
You're missing some escaping, see https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-ui/src/main/resources/XWiki/Realtime/Configuration.xml#L665 . We don't want the JSON to close the script tag by mistake.